PanCardView/PanCardView.csproj

```diff
@@ -50,6 +50,19 @@
     <Compile Include="SceneView.cs" />
     <Compile Include="Processors\BaseSceneBackProcessor.cs" />
     <Compile Include="Processors\BaseSceneFrontProcessor.cs" />
+    <Compile Include="Processors\MultiProcessors\BaseOpacityProcessor.cs" Condition=" '$(EnableDefaultCompileItems)' == 'true' " />

```

Comment:
Could you please remove Condition section?
Because it prevents building app on macbook

Reply:
Im on High Sierra with VS Mac 7.5.3 and builds alright :/